client/src/components/categorical/value/index.js

```diff
@@ -19,6 +19,8 @@ import Truncate from "../../util/truncate";
 
 import { AnnotationsHelpers } from "../../../util/stateManager";
 import { labelPrompt, isLabelErroneous } from "../labelUtil";
+import MiniHistogram from "../../expressionGraphs/miniHistogram";
```

Comment:
These charts are not specific to `expression`. Recommend top level `src/components/miniHistogram/index.js`

Reply:
@colinmegill, I created expressionGraphs to bundle the occupancy and mini histogram in the same folder.  Do you have a suggestion for a folder name that they would fit in?

Reply:
I named them expression graphs because they both show expression, whether that be of categorical or continuous data

Reply:
Moved to individual component folders